bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -1663,4 +1657,17 @@ public ByteBufAllocator getByteBufAllocator() {
     public ClientContext getClientCtx() {
         return clientCtx;
     }
+
+    private void checkLedgerCreationParameters(int ensSize, int writeQuorumSize, int ackQuorumSize) {
+        if (ensSize <= 0
+            || writeQuorumSize <= 0
+            || ackQuorumSize <= 0
```

Comment:
I think `ackQuorumSize==0`  is legal, which means no need ack; just write.

Reply:
I check again, `ackRuorumSize == 0` is meaningless for us. The `PendingAddOp#writeComplete` need to callback at least once. We didn't complete the `PendingAddOp` when the `ackRuorumSize == 0`.
So keep it as it is